Corona-Warn-App/src/main/res/layout/fragment_information_about.xml

```diff
@@ -24,7 +24,10 @@
             android:layout_width="0dp"
             android:layout_height="0dp"
             android:fillViewport="true"
-            app:layout_constraintBottom_toBottomOf="@+id/guideline_bottom"
+            android:paddingBottom="@dimen/spacing_small"
```

Comment:
I would align the bottom padding on this kind of fragments, I would go with `@dimen/spacing_normal` as it is here: https://github.com/corona-warn-app/cwa-app-android/pull/2884/files#diff-99108bde996a7dd32ce38fb5594a027efd59819f2b88d116dcc7b8f3eaced7ebR30